src/models/playlist.js

```diff
@@ -270,9 +270,16 @@ const updatePlaylistField = async (request, reply) => {
 
         const response = result.toJSON();
         const { ok, n } = response;
+        const userMessages = {
+            playlistDate: 'The playlist date was updated',
+            showId: 'This playlist has been reassigned to a different show',
+            noMatch: 'The playlist record has been updated'
+        };
+        const userMessage = userMessages[field] === undefined ?
+        userMessages.noMatch : userMessages[field];
 
         if (ok) {
-            return reply({ success: true, message: `${field} was updated` });
+            return reply({ success: true, message: userMessage });
```

Comment:
I still think that rather than use some conditional logic here to return a special message in this one case -- what happens when we need more special messages for more cases? -- let's instead extract this and add a generic mapping function from fields to messages instead. Feel free to ask questions! Also, seems like this PR does the same thing, so can it be closed? https://github.com/marciaga/kffp-admin/pull/36